web/web_test.go

```diff
@@ -68,11 +80,30 @@ func Setup() *TestHelper {
 	th := &TestHelper{
 		App:    a,
 		Server: s,
+		Web:    web,
 	}
 
 	return th
 }
 
+func (th *TestHelper) InitPlugins() *TestHelper {
```

Comment:
Could we leverage the `testlib` package and its resource management to handle this instead of doing it manually?